docs/quota_management.md

```diff
@@ -0,0 +1,102 @@
+---
+license: |
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+---
+
+Quota Management
+===
+
+Celeborn allows the administrator to set quotas for the number of names used.
+The administrator also can set a system level default quota for all user that 
+doesn't set a specified a named quota.
+
+When Master side sets `celeborn.quota.enabled` to true, the master will enable
+to check the quota via QuotaManager. When Client side also sets `celeborn.quota.enabled`
+to true, LifecycleManager will request master side to check whether the current user identifier
+has enough quota before registration of shuffle. Fallback to the default shuffle service
+of Spark when Master side checks that there is no enough quota for current user.
+
+## Quota
+Celeborn supports fine-grained quota management, now Celeborn including four indicators:
```

Comment:
[Quota](https://celeborn.apache.org/docs/latest/configuration/#quota) introduces the quota configs. Therefore this only adds the link of quota config.